client/src/containers/CompareContainer.js

```diff
@@ -27,6 +30,26 @@ function CompareContainer(props) {
     return catSettings.includes(o.espnId) && o.name !== 'mins'
   })
 
+  // Calculating Head to Head matchup table
+  if (!selectedTeams.includes('')) {
+    const h2h_row_1 = {}
+    const h2h_row_2 = {}
+    const teamOneId = parseInt(selectedTeams[0])
```

Comment:
I looked at this and saw the state values for the teamIds is actually strings which is non-ideal for the filtering. A quick cleanup would change `useState(['', '']);` to `useState([0, 0]);`  By handling it in the team change function, it doesn't need to be converted for every filtering operation.

and in the handleTeamChange `newSelectedTeams[position] = val;` to `newSelectedTeams[position] = parseInt(val);`

and line 67 ` if (row.teamId.toString() === team) {` to remove the toString